lib/time_for_a_boolean.rb

```diff
@@ -4,17 +4,22 @@
 require "time_for_a_boolean/railtie"
 
 module TimeForABoolean
-  def time_for_a_boolean(attribute)
+  def time_for_a_boolean(attribute, backing_attribute_name=nil)
+    backing_attribute_name ||= "#{attribute}_at"
```

Comment:
What do you think of `time_for_a_boolean(name, field="#{name}_at") for the signature and variable names?

```
2.2.3 :001 > def tfab(name, field="#{name}_at")
2.2.3 :002?>   puts name, field
2.2.3 :003?>   end
 => :tfab 
2.2.3 :004 > tfab "approved"
approved
approved_at
 => nil 
2.2.3 :005 > tfab "published", "published_on"
published
published_on
```